Data table style update

paulmatchett opened the pull request:
## Changes Made

- Updated the UI for search filtering by columns
- Updated table headers, active sorted column aces/desc and pagination styles.
- Fixed an issue on employee table were employeeDescription was not displaying any value

![image](https://user-images.githubusercontent.com/39831210/54025371-3a42d580-4192-11e9-8f74-340f87f95287.png)

paulmatchett committed:
tidied filter styles and created a SearchGroup styled component

paulmatchett committed:
tidied table styles and created a ReactTableContainer styled component

paulmatchett committed:
Fixed incorrect data entree for emplyee description

paulmatchett committed:
merged latest merges form develop

William-Unosquare reviewed (changes requested):
Looks good. Just a couple of things: 

I'm not so sure about the custom arrows in the select boxes, they look so unnecessary and also last time this was a thing in the internal app they were broken on Firefox if I recall correctly. 

You could improve this a bit more by removing "Search" in the header and in the placeholder use the language "Search by Status" and so on.

The reason I'm mainly requesting changes though is because you're exposing stuff like "clientName" in the placeholder text.

paulmatchett committed:
Updated placeholder value to equal select option label

paulmatchett committed:
fixed white space

paulmatchett commented:
Thanks for your review @William-Unosquare. The select styling is now the same everywhere on the app. I have just checked it in firefox and it is styling correctly. Your point about the input placeholder is spot on, so I have updated so that it displays the chosen select options label instead of value, and finally I have removed the label

paulmatchett committed:
removed label from imports

William-Unosquare reviewed (approved):
Looks good now 👍

paulmatchett committed:
Some folder updates
